### ISSUE TITLE
Create visual distinction between bike and car charging facilities amenity=charging_station

### ISSUE THREAD

**Issue by TheNewCivilian** [gravitystorm/openstreetmap-carto#4394]
### Expected behavior

For electric car / pedelec driver its important to display charging stations as already done by carto. In most cases of charging stations only one of those vehicle types can be charged. This can be very confusing. As there are a lot of charging stations designated for pedelec charging only (tagged with bicycle=yes and motorcar=no), depicting this distinction would be very helpful for both parties.
If there is interest in showing more specific icons here It might be a good idea to also support a new tagging schema using bicycle=designated for this new icon, as these charging points usually are fitted with regular power outlets not only supporting bike chargers, but are distinctively designated for pedelec charging

### Actual behavior

Currently there is only one icon used for charging stations on the map. For an electric car driver it can be confusing to see a e-bike charging station rendered with a e-fueling station icon and vice versa.

### Links and screenshots illustrating the problem

Therefore I propose a new Icon for bicycle charging points.

Before:
![Current generic charging station icon](https://user-images.githubusercontent.com/24456478/116375094-39a4a780-a80f-11eb-8b50-54ffa22e8a96.png)

After:
![Proposed pedelec charging Icon](https://user-images.githubusercontent.com/24456478/116375099-3b6e6b00-a80f-11eb-882e-22640e681974.png)

Actual Charging Point:
![Charging Point](https://chargemap.s3.amazonaws.com/charging_pool_photos/url/840x560/273427/1567338291_ios.jpeg)

**Comment by Adamant36** [comment]
I'd be interested to know what the difference is with the charging adaptors and other things are between the two. I know for the car charging facility down the road from where I live, it can also charge bikes and random USB devices, So, I wonder if there is similar overlap with bike charging stations. My guess is not because of something like lower voltages for charging bikes then cars, but who knows. 

The point is though, that even if something has a main thing it charges, that doesn't means it's all that can be charged there and  I think that should be considered when it comes to how the facilities are rendered or any changes made to said rendering.

**Comment by TheNewCivilian** [comment]
You are absolutely right, it exists a bright variety of charging points and configurations.

Public electric car chargers usually provide more than 4kW per socket. Pedelec chargers at least 400W. Other public charging points can be restricted to under 120W etc.

Still charging stations for pedelecs might be used for different devices as you pointed out. From my point of view that is as perfectly fine as bicycle_parking=shelter might be used as a storage box for buggies etc. too.
I would argue, if the designation is clear and visible in the real world, it should be visual on the map too.

This new visualization should prevent confusion by electric-car drivers, showing up at such chargers, while providing a more convincing symbol for cyclists.

The current charging point icon is quite car-centric and was submitted some years ago, based on a fueling station. Pedelec charging stations are a quite recent new development.

**Comment by TheNewCivilian** [comment]
I forked the repo added the icon and a simplified rule for testing. But I am struggeling to make it work O.o https://github.com/TheNewCivilian/openstreetmap-carto/tree/bike-charging-station Help needed!

**Comment by crse** [comment]
We should expand this to differentiate electric motorcycle charging as well.

There's plenty of them across Taiwan and SEA countries.

**Comment by imagico** [comment]
Current numbers:

* `amenity=charging_station`: 174k uses
* with `motorcar=yes`: 59k uses
* with `motorcycle=yes`: 8.9k uses
* with `bicycle=yes`: 8.6k uses

Would be helpful if someone would find out how many of the `motorcycle=yes`/`bicycle=yes` ones have functionally `motorcar=no` (either literally or with functionally equivalent tagging).
